Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1638,6 +1666,49 @@ namespace Multiplayer
         }
     }
 
+    void MultiplayerSystemComponent::MetricsEvent()
+    {
+        const auto& networkInterfaces = AZ::Interface<AzNetworking::INetworking>::Get()->GetNetworkInterfaces();
+        for (const auto& networkInterface : networkInterfaces)
+        {
+            if (networkInterface.second->GetType() != ProtocolType::Udp)
```

Comment:
Might want to have a CVAR around this so target protocol can be toggled.

Reply:
Done.